tests/unit/services/window-mock-test.js

```diff
@@ -140,3 +140,40 @@ test('it can call dispatchEvent', function(assert) {
   window.dispatchEvent(new Event('test-event'));
   assert.ok(spy.calledOnce);
 });
+
+test('it mocks window.localStorage', function(assert) {
```

Comment:
Can you maybe split these different test cases into separate tests? So when something is failing, it is easier to see what that is. Like separate tests for `.setItem()`/`.getItem()`, for `.length`, `.clear()`...

Btw, I should upgrade the test suite to the new ember-qunit APIs, so we can use nested `module`s, to group these tests together...

Reply:
Sure, done! All the individual tests are still dependent on at least one other method working, but hopefully this layout is clearer.